src/D2L.Bmx/BmxConfigProvider.cs

```diff
@@ -15,36 +12,42 @@ internal class BmxConfigProvider( FileIniDataParser parser ) : IBmxConfigProvide
 	public BmxConfig GetConfiguration() {
 		// Main config is at ~/.bmx/config
 		string configFileName = BmxPaths.CONFIG_FILE_NAME;
-
+		var data = new IniData();
+		if( File.Exists( configFileName ) ) {
+			try {
+				var tempdata = parser.ReadFile( configFileName );
+				data.Merge( tempdata );
+			} catch( Exception ) {
+				Console.Write( "Error occur while parsing config file." );
```

Comment:
* We rarely want to write to stdout, because that's reserved for `bmx print` to write the env var setting commands.
* nit: grammar and wording - I'm trying to make it clearer to the user that their config file is entirely ignored in this run. I'm a little worried that "error occurred" could be interpreted as "something wasn't quite right but it's mostly fine". Including the config file path is nice for troubleshooting
```suggestion
				Console.Error.Write( $"WARNING: Failed to load the global config file {configFileName}." );
```